src/main/java/seedu/address/logic/Messages.java

```diff
@@ -55,16 +56,26 @@ public static String format(Person person) {
      * Formats the {@code article} for display to the user.
      */
     public static String format(Article article) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm");
         final StringBuilder builder = new StringBuilder();
-        builder.append("; Title: ")
-                .append(article.getTitle())
-                .append("; Authors: ")
-                .append(article.getAuthors())
-                .append("; Publication Date: ")
-                .append(article.getPublicationDate())
-                .append("; Source: ")
-                .append(article.getSource())
-                .append("; Category: ")
+        builder.append(article.getTitle())
+                .append("; Authors: ");
+        for (int i = 0; i < article.getAuthors().length; i++) {
```

Comment:
I found it hard to understand what this code is doing, you can consider abstracting some code out as helper function, I think it would make this code more readable.